internal/resolver/resolver.go

```diff
@@ -107,6 +110,27 @@ func (r *Resolver) Resolve(reqs []*chart.Dependency, repoNames map[string]string
 			continue
 		}
 
+		if strings.HasPrefix(d.Repository, "git://") {
+
+			found, err := hasGitReference(strings.TrimPrefix(d.Repository, "git://"), d.Version, d.Name)
```

Comment:
It'd make this sort of code a bit nicer if the APIs in gitutils took an untrimmed URL, so that we don't have the explicit `"git://"` peppering the non-git-specific files more than necessary. (I see also a use of external `giturls.Parse` later, perhaps wrap that in a gitutils function that trims first and passes it through).